### PR TITLE
design-tokens: Added a prepublishOnly script

### PR DESCRIPTION
**Pull request by nathanstilwell**
Added a prepublishOnly script to ensure that tokens are built on package
publish.

### CODE REVIEW

**Review by laurenbarker — APPROVED** [review]
Ah, were they not built last time? I thought the diff looked a little weird, but I haven't confirmed it yet.

**Comment by laurenbarker** [comment]
It looked like the JS tokens were removed.

**Comment by nathanstilwell** [comment]
@laurenbarker Are you seeing that in the config or in the dist after building?

**Comment by laurenbarker** [comment]
@nathanstilwell the dist after building was what I was looking at, but I just looked again and I was just confused by a file that was removed. https://app.renovatebot.com/package-diff?name=@cockroachlabs%2fdesign-tokens&from=0.4.1&to=0.4.4#d2h-429753

`dist/web/tokens.module.js` was removed, but that's not JS tokens...

TLDR: you can ignore me lol

**Comment by nathanstilwell** [comment]
@laurenbarker Got it. 

I thought we had scripts for building all packages on publish, but I was wrong about `design-tokens`. @jocrl had a little hiccup publishing and didn't build manually before publishing `design-tokens@0.4.4`. I'm adding this script in and will publish `0.4.5` and do a little cleanup. This was a good exercise to see where we can make this process smoother.

**Comment by laurenbarker** [comment]
@nathanstilwell is there anything wrong with `design-tokens@0.4.4`? Should we wait to bump the version? It's included in the weekly renovate PR

**Comment by nathanstilwell** [comment]
@laurenbarker The only issue is that @jocrl renamed `stylusTokens.styl` to `tokens.styl` in the `config.json`, but version `0.4.4` was build without that change and so the `dist` folder contains a `stylusTokens.styl`. This shouldn't affect anything in CockroachCloud (which is where I assume you are running renovate) as nothing there consumes Stylus tokens.

**Comment by laurenbarker** [comment]
@nathanstilwell correct! Thanks for confirming!
